plugins/packages/packages_renderer.rb

```diff
@@ -26,6 +26,61 @@ def do_render
     end
   end
 
+  def do_render_comparison(description1, description2, description_common)
+    @changed_packages = if description1.packages && description2.packages
+      description1.packages.map(&:name) & description2.packages.map(&:name)
+    else
+      []
+    end
+
+    if description1.packages &&
+      !description1.packages.reject { |package| @changed_packages.include?(package.name) }.empty?
+      render_comparison_only_in(description1)
+    end
+    if description2.packages &&
+      !description2.packages.reject { |package| @changed_packages.include?(package.name) }.empty?
+      render_comparison_only_in(description2)
+    end
+    render_comparison_changed(description1, description2)
+    render_comparison_common(description_common) if @options[:show_all]
+  end
+
+  # In the comparison case we only want to show the package name, not all details like version,
+  # architecture etc.
+  def do_render_comparison_only_in
+    packages = @system_description.packages.reject { |package| @changed_packages.include?(package.name) }
+
+    list do
+      packages.each do |p|
+        item "#{p.name}"
+      end
+    end
+  end
+
+  def render_comparison_changed(description1, description2)
```

Comment:
Assignment Branch Condition size for render_comparison_changed is too high. [26.25/15]